Woof/Woof/Model/Sitter/Sitter+CodingKeys.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension Sitter {
+    // Defines custom CodingKeys enum to map properties to JSON keys
+    enum CodingKeys: String, CodingKey {
+        case id
+        case name
+        case surname
+        case phone
+        case avatarUrl
+        case bio
+        case rating
+        case pricePerHour = "rate_per_hour"
```

Comment:
The converting to snake case should be performed by `keyEncodingStrategy`.
```suggestion
        case pricePerHour = "ratePerHour"
```

Reply:
Fixed with a0069f9